WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -221,7 +235,6 @@ nick_the_ever_craving = {
 
 nick_the_anger_filled = {
 	allow = {
-		religion = buddhist
 		OR = {
```

Comment:
It's not an Eastern mentality. "Anger filled" is one of the Seven Deadly Sins (wroth) and Christians are called not to do it. A Christian with a hot temper can become famous for it.